src/cartesi-machine.lua

```diff
@@ -896,7 +891,7 @@ local options = {
             flash_length.root = nil
             flash_shared.root = nil
             table.remove(flash_label_order, 1)
-            bootargs = "quiet earlycon=sbi console=hvc0"
+            bootargs = bootargs:gsub(" rootfstype=.*$", "")
```

Comment:
Is this option always at the end of the bootargs when the option is triggered? It has to be the first option or something like this?

Reply:
All options after `rootfstype=` are related to the rootfs, the idea here is to clear out all rootfs ralted options from bootargs.

Reply:
These options are the absolute last thing you add to the bootargs, right? We should probably add a comment that this *has* to be the case, then.